README.md

```diff
@@ -1,4 +1,4 @@
-# gulp-eslint [![Build Status](https://travis-ci.org/adametry/gulp-eslint.svg)](https://travis-ci.org/adametry/gulp-eslint) [![Coverage Status](https://img.shields.io/coveralls/adametry/gulp-eslint.svg)](https://coveralls.io/r/adametry/gulp-eslint)
```

Comment:
OK it looks fixed now. It was broken yesterday for whatever reason, so I ignorantly assumed it was an outdated badge markup. Reverted that commit.